src/Clock.js

```diff
@@ -11,10 +11,14 @@ class Clock extends Component {
     // There is a bug in this component, where this.timer is not cleared if the
     // component is unmounted. It will continue to try to render if even the
     // component is no longer in the DOM.
-    this.timer = setTimeout(this.tick, 1000);
+    this.timer = setInterval(this.tick, 1000);
```

Comment:
Whoops, I totally missed that it was a timeout not an interval! I'll get that fixed for the next class :)